<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>noticky</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Clef:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="treble_button">
      <property name="text">
       <string>treble (G-clef)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="alto_button">
      <property name="text">
       <string>alto (C-clef)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="bass_button">
      <property name="text">
       <string>bass (F-clef)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>BPM</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="bpm_slider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>168</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="becko_button">
      <property name="text">
       <string>â™­</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="krizek_button">
      <property name="text">
       <string>#</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>0</y>
     <width>170</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Transposition:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="new_file_button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>New file</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>599</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QPushButton" name="c_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="d_button">
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="e_button">
      <property name="text">
       <string>E</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="f_button">
      <property name="text">
       <string>F</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="g_button">
      <property name="text">
       <string>G</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="a_button">
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="b_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="refresh_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openFile_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="musicEdit">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>129</y>
     <width>421</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>130</y>
     <width>431</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
